Templates/GitHubWebHook-CSharp/run.csx

```diff
@@ -0,0 +1,15 @@
+using System.Net;
+using System.Threading.Tasks;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+{
+    log.Verbose($"C# HTTP trigger function processed a request. RequestUri={req.RequestUri}");
+
+    // Get request body
+    dynamic data = await req.Content.ReadAsAsync<object>();
```

Comment:
We have a strongly typed object from ASP.NET WebHooks library. I think it would be better to use that instead of dynamic.